<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>756</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>323232</width>
    <height>323232</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STIX data parser and viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="viewer.qrc">
    <normaloff>:/Icons/images/app.svg</normaloff>:/Icons/images/app.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Packets</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QListView" name="listView">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>500</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </widget>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QTableWidget" name="tableWidget">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>2000</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
            <widget class="QTreeWidget" name="treeWidget">
             <attribute name="headerHighlightSections">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <property name="toolTip">
         <string>plot parameters
</string>
        </property>
        <attribute name="title">
         <string>Plot</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>244</y>
           <width>49</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string> Y axis: </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="paramNameEdit">
         <property name="geometry">
          <rect>
           <x>205</x>
           <y>244</y>
           <width>104</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>240</y>
           <width>86</width>
           <height>25</height>
          </rect>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>the same packet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>all packets</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>240</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Curve Style:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="styleEdit">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>240</y>
           <width>40</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QLabel" name="descLabel">
         <property name="geometry">
          <rect>
           <x>559</x>
           <y>244</y>
           <width>0</width>
           <height>17</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="plotButton">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>240</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
        <widget class="QPushButton" name="savePlotButton">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>240</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Save plot</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>244</y>
           <width>43</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>X axis:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="xaxisComboBox">
         <property name="geometry">
          <rect>
           <x>58</x>
           <y>244</y>
           <width>86</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Parameter order</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Timestamp</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>240</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>In</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QDockWidget" name="dockWidget_2">
       <property name="windowTitle">
        <string>Log</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents_2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="listWidget_2"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSet_IDB"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Plot"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuAction"/>
   <addaction name="menuSetting"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="action_Plot"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionSet_IDB">
   <property name="text">
    <string>Set &amp;IDB</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Sa&amp;ve</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="action_Plot">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/Icons/images/graph.svg</normaloff>:/Icons/images/graph.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
